examples/plot_visualize_bad_epochs.py

```diff
@@ -131,25 +114,25 @@
 plt.colorbar()
 plt.show()
 
-# %%
+###############################################################################
 # ... and visualize the bad epochs and sensors. Bad sensors which have been
 # interpolated are in blue. Bad sensors which are not interpolated are in red.
 # Bad trials are also in red.
 
 scalings = dict(eeg=40e-6)
 reject_log.plot_epochs(this_epoch, scalings=scalings)
 
-# %%
+###############################################################################
 # ... and the epochs after cleaning with autoreject
 
 epochs_ar.plot(scalings=scalings)
 
-# %%
+###############################################################################
```

Comment:
@alexrockhill please don't touch files that are not related to PR. It will make review and merge much slower even if we're best friends ;-) 

anyhow, don't touch now. I'll do my pass

Reply:
@alexrockhill also: why change from `# %%` to `###############################` ?

`# %%` makes it possible to have the file in "notebook" style in VS code and other editors. See https://sphinx-gallery.github.io/stable/syntax.html#embed-rst-in-your-example-python-files

> The #%% and # %% syntax is consistent with the ‘code block’ (or ‘code cell’) separator syntax in Visual Studio Code Python extension, Visual Studio Python Tools, Jupytext, Pycharm Professional, Hydrogen plugin (for Atom) and Spyder.

what would the benefit of `#####################` be?

Reply:
I didn't have a reason to prefer either, the examples are just inconsistent in style and I've seen ###### in mne more often